src/components/TheConnectingDialog.vue

```diff
@@ -28,7 +28,7 @@
                 </div>
             </v-card-text>
             <v-card-text v-else class="pt-5">
-                <v-progress-linear color="white" indeterminate></v-progress-linear>
```

Comment:
Please only remove this only in light mode. So right now, this progress-line will displayed in primary color. at start up, it will be everytime blue (because the gui settings are loaded later). i would prefer a gray color in light mode and white in dark mode.

for always light mode, we also need a setting in the config.json.